ui/core/components/individual_sim_ui/apl_helpers.ts

```diff
@@ -33,8 +33,9 @@ const actionIdSets: Record<
 		defaultLabel: 'Aura',
 		getActionIDs: async metadata => {
 			return metadata.getAuras().map(actionId => {
-				const hasRanks =
-					metadata.getAuras().filter(mainSpell => mainSpell.id.name.startsWith(actionId.id.name.replace(/ \(Rank \d+\)/g, ''))).length > 1;
+				const baseActionName = actionId.id.name.replace(/ \(Rank \d+\)/g, '');
+				const rankedNameRegex = new RegExp(`${baseActionName} \\(Rank [0-9]+\\)`);
+				const hasRanks = metadata.getAuras().filter(spell => !!spell.id.name.match(rankedNameRegex)).length > 1;
```

Comment:
These changes group ranked spells without grouping Sunfire and Sunfire (Cat) which is what I wanted